src/components/header/Header.test.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 import Header from './Header';
-import { render, cleanup, fireEvent } from '@testing-library/react'
+import { render, cleanup, fireEvent, getByAltText } from '@testing-library/react'
 
 describe('Header', () => {
     afterEach(() => {
         cleanup();
     })
 
     it('renders Authenticated Header if user exists', () => {
-        const component = render(
-            <Header user={{}} />
-        )
-        
-        component.getByText('Sell')
-        component.getByText('Log Out')
+        const { getByAltText, getByText } = render(<Header user={{}} />)
+        fireEvent.click(getByAltText('avatar'))
+
+        getByText('Sell')
```

Comment:
I'm not familiar with this testing library - how does the `getBy` function work? Does it just return the matching element based on the passed in text? Why don't you have some sort of assertion/expect here? Like `expect(getByText(testMessage)).toBeInTheDocument()`

Reply:
The getByText itself is an assertion -- it asserts that the text "Sell" is in the DOM tree